driver-core/src/test/resources/initial-dns-seedlist-discovery/README.rst

```diff
@@ -9,7 +9,8 @@ Test Setup
 ----------
 
 Start a three-node replica set on localhost, on ports 27017, 27018, and 27019,
-with replica set name "repl0".
+with replica set name "repl0". The replica set MUST be started with SSL
```

Comment:
Do we need this file in the resources?

Reply:
It's not essential but it does explain what's going on.  Without knowing what's in the SRV/TXT DNS records the tests don't make sense.